heltour/tournament/admin.py

```diff
@@ -157,42 +273,45 @@ class SeasonAdmin(_BaseAdmin):
+                self.admin_site.admin_view(self.manage_players_view),
                 name='manage_players'),
             url(r'^(?P<object_id>[0-9]+)/player_info/(?P<player_name>[\w-]+)/$',
-                permission_required('tournament.manage_players')(self.admin_site.admin_view(self.player_info_view)),
+                self.admin_site.admin_view(self.player_info_view),
                 name='edit_rosters_player_info'),
             url(r'^(?P<object_id>[0-9]+)/round_transition/$',
-                permission_required('tournament.generate_pairings')(self.admin_site.admin_view(self.round_transition_view)),
+                self.admin_site.admin_view(self.round_transition_view),
                 name='round_transition'),
             url(r'^(?P<object_id>[0-9]+)/review_nominated_games/$',
-                permission_required('tournament.review_nominated_games')(self.admin_site.admin_view(self.review_nominated_games_view)),
+                self.admin_site.admin_view(self.review_nominated_games_view),
                 name='review_nominated_games'),
             url(r'^(?P<object_id>[0-9]+)/review_nominated_games/select/(?P<nom_id>[0-9]+)/$',
-                permission_required('tournament.review_nominated_games')(self.admin_site.admin_view(self.review_nominated_games_select_view)),
+                self.admin_site.admin_view(self.review_nominated_games_select_view),
                 name='review_nominated_games_select'),
             url(r'^(?P<object_id>[0-9]+)/review_nominated_games/deselect/(?P<sel_id>[0-9]+)/$',
-                permission_required('tournament.review_nominated_games')(self.admin_site.admin_view(self.review_nominated_games_deselect_view)),
+                self.admin_site.admin_view(self.review_nominated_games_deselect_view),
                 name='review_nominated_games_deselect'),
             url(r'^(?P<object_id>[0-9]+)/review_nominated_games/pgn/$',
-                permission_required('tournament.review_nominated_games')(self.admin_site.admin_view(self.review_nominated_games_pgn_view)),
+                self.admin_site.admin_view(self.review_nominated_games_pgn_view),
                 name='review_nominated_games_pgn'),
             url(r'^(?P<object_id>[0-9]+)/bulk_email/$',
-                permission_required('tournament.bulk_email')(self.admin_site.admin_view(self.bulk_email_view)),
+                self.admin_site.admin_view(self.bulk_email_view),
                 name='bulk_email'),
             url(r'^(?P<object_id>[0-9]+)/mod_report/$',
-                permission_required('tournament.change_season')(self.admin_site.admin_view(self.mod_report_view)),
+                self.admin_site.admin_view(self.mod_report_view),
                 name='mod_report'),
         ]
         return my_urls + urls
 
 
     def simulate_tournament(self, request, queryset):
+        if not request.user.is_superuser:
```

Comment:
I'm probably going to remove this code entirely, it's more appropriate for a separate script